client/src/components/relatedProducts/test/modal.test.js

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable no-unused-vars */
+import React from 'react';
+import { render } from '@testing-library/react';
+import Modal from '../Modal.jsx';
+
+describe('modal', () => {
+  const currentProduct = {
```

Comment:
feel free to import from `testProduct` or add more tailored mock data, which may save a few lines for the test file